src/http/index.js

```diff
@@ -1,14 +1,8 @@
-import qs from 'qs';
-import jsYaml from 'js-yaml';
-
 import '../helpers/fetch-polyfill.node.js';
-import { valueEncoder } from '../execute/oas3/style-serializer.js';
+import { serializeRequest } from './serializers/request/index.js';
+import { serializeResponse, serializeHeaders } from './serializers/response/index.js';
 
-// For testing
-export const self = {
-  serializeRes,
-  mergeInQueryOrForm,
-};
+export { serializeResponse as serializeRes, serializeHeaders };
```

Comment:
Left exporting these here as we do expose them:
https://github.com/swagger-api/swagger-js/blob/df80732d2a1c4f2e82bc3a10e8fd8cb1ffc7be64/src/index.js#L3
https://github.com/swagger-api/swagger-js/blob/df80732d2a1c4f2e82bc3a10e8fd8cb1ffc7be64/src/index.js#L46-L47
and serializeRes is also imported in Swagger UI:
https://github.com/swagger-api/swagger-ui/blob/f7373a0bc11268e266eca2173bb143d0eb263a32/src/core/plugins/swagger-client/index.js#L7
https://github.com/swagger-api/swagger-ui/blob/f7373a0bc11268e266eca2173bb143d0eb263a32/src/core/plugins/swagger-client/index.js#L43

although I don't see any other usage for it there.

Reply:
Let's not export `serializeHeaders` from here. If needed by other code, it should import from `http/serializers/response/**.js`.